index.html

```diff
@@ -3,27 +3,42 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
     <link rel="stylesheet" href="css/reset.css" />
     <link rel="stylesheet" href="styles.css" />
+    <script src="./js/index.js" defer></script>
     <title>IsaMarvin | Software Engineer</title>
   </head>
   <body>
     <header>
       <nav>
         <div class="logo">
           <a href="#">
-            <img src="./images/my-logo.svg" alt="" />
+            <img src="./images/my-logo.svg" alt="Logo" />
           </a>
         </div>
```

Comment:
- [ Optional] I recommend using your own logo or name, it will look more professional.